testing/database.rst

```diff
@@ -48,12 +48,12 @@ Now, enable it as a PHPUnit extension or listener:
     <phpunit>
         <!-- ... -->
 
-        <!-- Add this in PHPUnit 8 or higher -->
+        <!-- Add this for PHPUnit 7.1 or higher -->
         <extensions>
             <extension class="DAMA\DoctrineTestBundle\PHPUnit\PHPUnitExtension"/>
         </extensions>
 
-        <!-- Add this in PHPUnit 7 or lower -->
+        <!-- Add this for PHPUnit 7.0 or lower -->
```

Comment:
actually current version 6 does not support all phpunit 6.x versions anymore. So officially it only supports PHPUnit 7 and 8.

Should we somehow mention this here?

Reply:
Ok actually its incompatible with all PHPUnit versions below 7.0.

Even 6.5 does not work:

```
In PHPUnitListener.php line 7:
                                                                                                                                                                                                                  
  [Symfony\Component\Debug\Exception\FatalErrorException]                                                                                                                                                         
  Compile Error: Declaration of DAMA\DoctrineTestBundle\PHPUnit\PHPUnitListener::endTest(PHPUnit\Framework\Test $test, float $time): void must be compatible with PHPUnit\Framework\TestListener::endTest(PHPUni  
  t\Framework\Test $test, $time)                                                                                                                                                                                  
                                                                                                                                                                                                                  

Exception trace:
  at /var/www/doctrine-test-bundle/src/DAMA/DoctrineTestBundle/PHPUnit/PHPUnitListener.php:7
```

But PHPUnit 6 is not supported anymore officially anyway :yum:

Reply:
Yes, I think it's OK to ignore PHPUnit 6 nowadays.